test-vfkit/vfkit_suite_test.go

```diff
@@ -86,9 +88,17 @@ var _ = ginkgo.BeforeSuite(func() {
 outer:
 	for panics := 0; ; panics++ {
 		_ = os.Remove(sock)
-
-		// #nosec
-		host = exec.Command(filepath.Join(binDir, "gvproxy"), fmt.Sprintf("--ssh-port=%d", sshPort), fmt.Sprintf("--listen=unix://%s", sock), fmt.Sprintf("--listen-vfkit=unixgram://%s", vfkitSock))
+		_ = os.Remove(vfkitSock)
```

Comment:
I wonder if we should remove the `_ = os.Remove(sock)` and `_ = os.Remove(vfkitSock)` in favor of `clear()`